modules/monitoring-granting-users-permission-to-monitor-user-defined-projects.adoc

```diff
@@ -11,7 +11,7 @@ Cluster administrators can grant developers and other users permission to monito
 
 * The *monitoring-rules-view* cluster role provides read access to `PrometheusRule` custom resources for a project.
 
-* The *monitoring-rules-edit* cluster role grants a user permission to create, modify, and deleting `PrometheusRule` custom resources for a project.
+* The *monitoring-rules-edit* cluster role grants a user permission to create, modify, and delete `PrometheusRule` custom resources for a project. It also grants a user the ability to silence alerts.
```

Comment:
@eromanova97 Nice work! I do see instances where you must [avoid anthropomorphism](https://docs.engineering.redhat.com/display/CCSENABLE/Anthropomorphism). For example, "The monitoring-rules-edit cluster role grants a user permission.."

I will not hold the merge. But please address this comment for all such instances in another JIRA. Thank you!

Reply:
Thank you @Srivaralakshmi for noticing this. I will definitely fix the missing module metadata.
Regarding the anthropomorphism, I am wondering if we want to change this whole module because of it, because IBM guide does mention one exception:
"... unless the object is a type of authorization that does grant abilities."
I think the cluster role would fall under this category, but I will also talk about this with Brian to discuss. Thank you!